tests/end_to_end/test_target_postgres.py

```diff
@@ -1,13 +1,19 @@
 import os
+import uuid
+from datetime import datetime
+from random import randint
+
+import bson
 import pytest
+from bson import Timestamp
 
 from .helpers import tasks
 from .helpers import assertions
 from .helpers.env import E2EEnv
 
 DIR = os.path.dirname(__file__)
 TAP_MARIADB_ID = 'mariadb_to_pg'
-TAP_MONGODB_ID = 'tap_mongo'
+TAP_MONGODB_ID = 'tap_mongo_to_pg'
```

Comment:
could we make this `mongo_to_pg` to keep with the convention

Reply:
fixed in [c88330af5082a89b11a83ef0ab6984dbadd86afb](https://github.com/transferwise/pipelinewise/pull/434/commits/c88330af5082a89b11a83ef0ab6984dbadd86afb).